packages/fxa-auth-server/lib/senders/emails/templates/fraudulentAccountDeletion/en.ftl

```diff
@@ -1,7 +1,10 @@
 fraudulentAccountDeletion-subject = Your { -product-firefox-account } was deleted
+fraudulentAccountDeletion-subject-2 = Your { -product-mozilla-account } was deleted
 fraudulentAccountDeletion-title = Your account was deleted
 fraudulentAccountDeletion-content = Recently, a { -product-firefox-account } was created and a subscription was charged using this email address. As we do with all new accounts, we asked that you confirm your account by first validating this email address.
+fraudulentAccountDeletion-content-moz = Recently, a { -product-mozilla-account } was created and a subscription was charged using this email address. As we do with all new accounts, we asked that you confirm your account by first validating this email address.
```

Comment:
```suggestion
fraudulentAccountDeletion-content-2 = Recently, a { -product-mozilla-account } was created and a subscription was charged using this email address. As we do with all new accounts, we asked that you confirm your account by first validating this email address.
```

Nit: only this and one other string are updated using `-moz`. Suggestion is for consistency.

Reply:
This one was tricky, because `fraudulentAccountDeletion-content`and `fraudulentAccountDeletion-content-2` are both currently in use, and not different versions of the same text. Perhaps the new version could be `fraudulentAccountDeletion-content-part1-v2` and `fraudulentAccountDeletion-content-part2-v2`?